core/src/main/java/org/testcontainers/dockerclient/DockerClientProviderStrategy.java

```diff
@@ -145,7 +146,7 @@ public static DockerClientProviderStrategy getFirstValidStrategy(List<DockerClie
 
                         Info info;
                         try {
-                            info = Unreliables.retryUntilSuccess(30, TimeUnit.SECONDS, () -> {
+                            info = Unreliables.retryUntilSuccess(5, TimeUnit.SECONDS, () -> {
```

Comment:
nit: maybe it would be worth it to extract that value as configuration property
@bsideup @kiview @rnorth  whats your view on that ?

Reply:
This may not be a bad idea, actually 👍

Reply:
Perhaps, besides making the default timeout much shorter, we could also make it `config`urable (again)? So for people who need different TO (due to whatever reason) the would be an option?